sunbeamlib/__init__.py

```diff
@@ -90,3 +91,13 @@ def circular(seq, kmin, kmax, min_len):
             return True
     return False
 
+def read_seq_ids(fasta_fp):
```

Comment:
I know it's short and simple, but I'd prefer we use pre-written FASTA parsers from BioPython to handle any edge cases. BioPython is already a requirement so it's already available:

```python
from Bio import SeqIO
def read_seq_ids(fasta_fp):
    return [record.id for record in SeqIO.parse(fasta_fp, "fasta")]
```